packages/frontend/store/system/actions.ts

```diff
@@ -1,14 +1,28 @@
 import {
   SET_MOBILE_VIEW,
+  SET_WINDOW_DIMENSIONS,
   ISystemState,
-  ISetMobileViewAction
+  ISetMobileViewAction,
+  ISetWindowDimensionsAction,
+  Width,
+  Height,
 } from './types';
 
-const setMobileView = (mobileView: ISystemState['mobileView']): ISetMobileViewAction => {
+export const setMobileView = (
+  mobileView: ISystemState['mobileView'],
+): ISetMobileViewAction => {
   return {
     type: SET_MOBILE_VIEW,
     mobileView,
   };
 };
 
-export default setMobileView;
+export const setWindowDimensions = (
+  width: Width,
+  height: Height,
```

Comment:
Let's given in a WindowDimension object here instead of two separate numbers.

Reply:
What if we need the type for width or height one day? Should we do setWindowDimensions.width?

Reply:
![image](https://user-images.githubusercontent.com/42633156/76057030-3e374c80-5fcc-11ea-8ef4-71cf91dc20d6.png)
Can do something like this :P